packages/yew-macro/src/html_tree/html_element.rs

```diff
@@ -362,6 +361,10 @@ impl ToTokens for HtmlElement {
                 // this way we get a nice error message (with the correct span) when the expression
                 // doesn't return a valid value
                 quote_spanned! {expr.span()=> {
+                    #[allow(unused_braces)]
+                    // e.g. html!{<@{"div"}/>} will set `#expr` to `{"div"}`
+                    // (note the extra braces). Hence the need for the `allow`.
+                    // Anyways to remove the braces?
                     let mut #vtag_name = ::std::convert::Into::<
                         ::std::borrow::Cow::<'static, ::std::primitive::str>
                     >::into(#expr);
```

Comment:
This is where the misplaced `#[allow]` really belongs. Maybe the `allow` here is also liftable? I'm however too inexperienced with proc macro to figure it how. So I just left some comments instead